.github/workflows/archive/documentationcheck.py

```
@@ -0,0 +1,222 @@
+    )
+    # getting root directory of repository
+    parser.add_argument(
+        "--directory",
+        type=str,
+        required=False,
+        default=os.getcwd(),
+        help="The parent directory of files to analyze.",
+    )
+    # Return parser
+    result = parser.parse_args()
+    return result
```

Ruff (0.11.9)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)